addons/html_editor/models/ir_qweb_fields.py

```diff
@@ -66,13 +80,13 @@ def _compile_node(self, el, compile_context, level):
                         el_children[0].set('t-options', f"{{'snippet-key': '{snippet_key}', 'snippet-sub-call-key': '{sub_call}'}}")
         # If it already has a data-snippet it is a saved or an
         # inherited snippet. Do not override it.
-        if 'data-snippet' not in snippet_base_node.attrib:
+        if snippet_base_node.tag != 't' and 'data-snippet' not in snippet_base_node.attrib:
```

Comment:
I don't think we had snippets with a `<t>` tag, but are you adding this check to make sure we don't add `data-snippet` on a `<t>` (like what you are doing in this PR) ?